run_dir/static/js/project_samples.js

```diff
@@ -787,6 +788,16 @@ function load_samples_table(colOrder) {
             $.each(libs, function(idx, library){
               info_library=info['library_prep'][library];
               if ('library_validation' in info_library) {
+                // Populate the library_validation object when undefined
+                if (Object.keys(info_library['library_validation']).length === 0) {
+                  info_library['library_validation']['-'] = { 'average_size_bp': "-", 'conc_units': "-", 'concentration': "-", 'finish_date': "-", 'initials': "-", 'prep_status': "-", 'size_(bp)': "-", 'start_date': "-", 'volume_(ul)': "-", 'well_location': "-"};
+                  // Populate additional empty fields
+                  if (!(info['prep_status'].length === 0 || info['prep_status'] == '-' && info['initial_qc']['initial_qc_status'] === undefined || info['initial_qc']['initial_qc_status'] == '-' && info['initial_qc']['failure_reason'] === undefined || info['initial_qc']['failure_reason'] == '-')){
```

Comment:
Would this be empty only if `info_library['library_validation']` is empty? `info['prep_status']` and `info['library_prep'][library]['library_validation']` seem to be different things. If not, then this if condition should be outside` if (Object.keys(info_library['library_validation']).length === 0)`

Reply:
From looking at a few projects it seems like the problems are caused by a prep that might have gotten aborted, and therefore the `info_library['library_validation']` is an empty object. 
When this is empty, the prep status is also empty (for the prep). 

But I have asked the lab for more input, to sort out what is causing the mess 🤷‍♀️